practice/00-HTML-CSS-basics/botom.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Form</title>
+</head>
+<body>
+<div>
+	<h1>sign up</h1>
+	<form class="contact_form">
+		<ul>
+				<li><label for="name">Name: </label><input type="text" placeholder="Michael" required></li>
```

Comment:
Your input element must have an id if you want to use the for attribute in the label. The content of the for attribute must be equal to the id of the input tag.